### ISSUE TITLE
fix build please

### ISSUE THREAD

**Issue by ildar** [fedefernandez/MyAppList#13]
Hi! I really like the app even in 2014-year condition :)

I want it back to F-Droid but it cannot build. Could you please look at https://gitlab.com/fdroid/fdroiddata/issues/944 ? thanks in advance.

**Comment by fedefernandez** [comment]
I'm sorry but it's too outdated. I'll try to refresh the libraries but I can't promise anything, a lot of stuff is just missing for the latest java/android versions

**Comment by ildar** [comment]
sure. Thanks.

If it helps, someone already tried:
https://github.com/bitliuxm/MyAppList/commit/94e7c959e198376295e26a27ca81f98fa9afa0dc

**Comment by cauerego** [comment]
amazing how this app is still useful today and better than any other i've found so far!

**Comment by fedefernandez** [comment]
Yes, I'm surprised too. I'm finding too many issues trying to replace the multi-select list, specially now I'm not an android developer
